src/openforms/registrations/tasks.py

```diff
@@ -103,6 +105,13 @@ def register_submission(submission_id: int) -> None:
         # if it's already successfully registered, do not overwrite that.
         return
 
+    if submission.waiting_on_cosign:
+        logger.debug(
+            "Skipping registration for submission '%s' as it hasn't been co-signed yet.",
+            submission,
+        )
+        return
```

Comment:
Perhaps we should also have a `logevent` record here to show in the admin for a given submission?